### DISCUSSION TITLE
Fill the left side of the page with the post.

### DISCUSSION THREAD

**Discussion by Koushikphy** [mmistakes/minimal-mistakes#2638] [Q&A] [answered]
This is the page in question : https://koushikphy.github.io/ide_quick_refrence/
This the page in the repo: https://github.com/Koushikphy/Koushikphy.github.io/blob/master/_pages/ide_ref.md
### Problem: 
1. As you can see the page has a blank on the left side, where the author profile stays. I want to fill the place with the post but I can't get it done. I don't want the author profile but I want to keep the table of contents on the right side. Is there a way to do that?
2. I want to keep the table of contents `sticky`, so as the user scrolls the page the `toc` keeps in view. But the `toc` is long and if I make it `sticky` then the lower side of the `toc` becomes completely blocked from view. So can I make the `toc` so that instead of being completely `sticky` it scrolls up a little when the user scrolls to the bottom of the page.

**Comment by iBug** [comment, marked as answer]
Duplicate of #2453
